exec/graph.go

```diff
@@ -55,24 +55,24 @@ func (s *Session) handleTasksGraph(w http.ResponseWriter, r *http.Request) {
 		indexed[task] = len(indexed)
 	}
 
-	type node struct {
+	type Node struct {
```

Comment:
comment these exported structs?

Reply:
They are scoped to this function, so I assert that it's not necessary or useful. I capitalized them to avoid the shadowing, e.g. `var node node`. Alternatively, I could name the variable `n`, but I thought that was more confusing.

Reply:
oh, yeah. that's seems fine.